VKApp/VKApp/VKAPI.swift

```diff
@@ -21,7 +23,105 @@ final class VKAPIService {
         }
         let url = Constants.baseURLText + Constants.methodText + method
```

Comment:
```suggestion
        let url = Constants.baseURLText + Constants.methodText + method
```
переделать на интерполяцию